examples/nvidia_cuda_q/container/Dockerfile

```diff
@@ -1,17 +1,16 @@
-FROM 292282985366.dkr.ecr.us-west-2.amazonaws.com/amazon-braket-pytorch-jobs:latest
-RUN python3 -m pip install --upgrade pip
+FROM 292282985366.dkr.ecr.us-west-2.amazonaws.com/amazon-braket-base-jobs:latest
 
-# install cudaq
 ARG SCRIPT_PATH
-ARG CUDAQ_PATH=/opt/conda/lib/python3.10/site-packages
-ENV MPI_PATH=/opt/amazon/openmpi
 
-RUN python3 -m pip install cudaq
-RUN bash "${CUDAQ_PATH}/distributed_interfaces/activate_custom_mpi.sh"
+# install cudaq
+ENV MPI_PATH=/usr/local
```

Comment:
```suggestion
ENV MPI_PATH=/usr/local \
  SAGEMAKER_PROGRAM=braket_container.py
```

Reply:
This is more a style choice. Letting all the ENV variables get grouped at the top is easier to reference back and forth for me rather than searching in line for their definition..